exchangelib/items/calendar_item.py

```diff
@@ -366,11 +366,11 @@ class BaseMeetingReplyItem(BaseItem, metaclass=EWSMeta):
     proposed_end = DateTimeField(field_uri='meeting:ProposedEnd', supported_from=EXCHANGE_2013)
 
     @require_account
-    def send(self, message_disposition=SEND_AND_SAVE_COPY):
+    def send(self, message_disposition=None):
         return CreateItem(account=self.account).get(
```

Comment:
Changing the default value is not necessary here.

Reply:
You are right. This is an old reflex to set None as default to avoid mutable value in method signature. But SEND_AND_SAVE_COPY is immutable. I will update ASAP